languages/elixir/exercises/concept/strings/.docs/hints.md

```diff
@@ -0,0 +1,41 @@
+### General
+
+- Read about strings in the official [Getting Started Elixir guide][getting-started-strings].
+- Browse the [functions available in the _String module_][string-module-functions] to discover which operations on strings Elixir's standard library offers.
+
+### 1. Get the name's first letter
+
+- There are a few different ways to get a specific character or characters from a string based on their index: [`String.at/2`][string-at], [`String.slice/3`][string-slice], [`String.first/1`][string-first], [`String.last/1`][string-last]
```

Comment:
I think I might not have expressed myself well in my last comment. Your previous hints were really nice, but they spelled out the solution. In other words: they contained direct code/function references as text. 

The previous version looked like:

>  Use [`String.first/1`][string-first] to get the first character from a string.

What I wanted to suggest was to do:

>  There is a [built-in function][string-first] to get the first character from a string.

Or something similar. This version does link to the correct function, which means that students will likely be able to un-block themselves, but doesn't provide the code for the actual solution. This means that they'll have to go to the link you've provided them, which is something we want to encourage as knowing where to find a language's documentation and how to read it is a very important skill. But as said, this can be changed in a future PR :)

Reply:
Now I understand, thanks!